deployment/helm/synopsys-alert/README.md

```diff
@@ -196,6 +199,68 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 $ helm install . --name <name> --namespace <namespace> --set enableStandalone=true
 ```
 
+### Database Credential Secrets
+This section describes how to configure Alert using kubernetes secrets to set the database user and administrator credentials.
+
+#### Create the Database User Credentials Secret
+- Execute the command
+  ```bash 
+   $ kubectl create secret generic <SECRET_NAME> -n <ALERT_NAMESPACE> \
+   --from-file=ALERT_DB_USERNAME=<USER_USERNAME> \
+   --from-file=ALERT_DB_PASSWORD=<USER_PASSWORD>
+  ```
+  - Replace `<SECRET_NAME>` with the desired name for the secret.
+  - Replace `<ALERT_NAMESPACE>` with the namespace being used for Alert.
+  - Replace `<USER_USERNAME>` with the database user owning the Alert database.
+  - Replace `<USER_PASSWORD>` with the database user's password.
+  - Note: Using this secret will take priority over setting userUserName and userPassword in the Configuration Parameters in values.yaml.
```

Comment:
Is `userUserName` a typo?

Reply:
no, that is correct. In the configuration file we have both userUserName referring the db user, and another adminUserName for the postgres admin.